test/integration/user_login_test.rb

```diff
@@ -2,14 +2,18 @@
 
 class UserLoginTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
+  attr_reader :user
 
-  test "user can login" do
-    user = User.create(username: 'user',
-                       password: 'password',
-                       first_name: 'John',
-                       last_name: 'Doe',
-                       email: 'example@example.com')
+  def setup
+    @user = User.create(username: 'user',
+                      password: 'password',
+                      first_name: 'John',
+                      last_name: 'Doe',
+                      email: 'example@example.com')
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.